packages/css-handler/package.json

```diff
@@ -0,0 +1,8 @@
+{
+  "name": "css-handler",
+  "bin": "src/generate.mjs",
```

Comment:
Important! This assigns the executable for the package when installed into other packages. You'll see this get executed as `css-handler` in scripts in other packages.